packages/webapp/src/views/credits-purchase/check-and-submit.html

```diff
@@ -21,6 +21,7 @@
     'valueId': 'metric-development-details-confirmed-value',
     'href': metric.detailsConfirmedUrl,
     'visuallyHiddenText': ' confirm development details',
+    'actionClasses':'hidden',
```

Comment:
You should use the newly added`'noAction': true` option that stops the link being added within the common template rather than just hiding with CSS